Paintroid/src/main/res/layout/dialog_pocketpaint_shapes.xml

```diff
@@ -17,8 +17,98 @@
  *  You should have received a copy of the GNU Affero General Public License
  *  along with this program.  If not, see <http://www.gnu.org/licenses/>.
  -->
-<merge xmlns:android="http://schemas.android.com/apk/res/android">
+<merge xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:tools="http://schemas.android.com/tools">
 
+    <TextView
+        android:id="@+id/pocketpaint_outline_view_text_view"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:layout_marginBottom="4dp"
+        android:text="@string/dialog_brush_width_text"
+        android:textAllCaps="true"
+        android:textColor="@color/pocketpaint_main_rectangle_tool_accent_color"
+        android:textStyle="bold"
+        android:layout_marginEnd="16dp"
+        android:layout_marginStart="24dip" />
+
+    <View
+        style="@style/PocketPaintDividerHorizontal"
+        android:id="@+id/pocketpaint_outline_view_border" />
+
+    <LinearLayout
+        android:layout_width="match_parent"
+        android:layout_height="match_parent"
+        android:weightSum="5"
+        android:orientation="horizontal" >
+
+        <SeekBar
+            android:id="@+id/pocketpaint_shape_stroke_width_seek_bar"
+            android:progressTint="@color/pocketpaint_colorAccent"
```

Comment:
I think you can just remove this, the generated `AppCompatSeekBar` should use `colorAccent` automatically.